roles/kubernetes/preinstall/tasks/0020-set_facts.yml

```diff
@@ -7,6 +7,10 @@
   tags:
     - facts
 
+- name: Set OS families based on distribution
+  set_fact:
+    os_families: "{{ 'openEuler' if ansible_distribution == 'openEuler' else ansible_os_family }}"
```

Comment:
what's the goal of setting a new variable here ?  


os_families = ansible_distribution  when it's openEuler


why not directly use ansible_distribution == 'openEuler' when you need to know ?